### DISCUSSION TITLE
history of commands

### DISCUSSION THREAD

**Discussion by hinxx** [OpenBB-finance/OpenBB#257] [General]
I'm used to having access to the history of commands while using terminal in Linux. Pressing UP arrow shows the last command executed. It may be that I'm just lazy in typing but what are is the probability in having this in GST?

**Comment by DidierRLopes** [comment]
I think what you want already exists.

Have a look at this: https://github.com/DidierRLopes/GamestonkTerminal/blob/3711148ba5e2633b212d9d6f8fc681974499a069/gamestonk_terminal/feature_flags.py#L14

This feature flags allows to enable/disable several features that may be down to user preference. That's one of them, and as you can see, it defaults to False currently. We probably can default it to True in the future to be fair.

**Reply by hinxx** [reply]
Indeed what I was missing. Also has the <TAB> completion, wow.. laziness ^ 2!

**Reply by DidierRLopes** [reply]
It's a cool one. I only found out about it because of a collaborator! Pretty sick!
